modules/__init__.py

```diff
@@ -1,19 +1,20 @@
-from modules.base import Module, noparts, check_solve_cmd, gif_append, gif_output
 from os.path import basename, dirname, join as path_join
 from glob import glob
 import importlib
 import inspect
 
+LIST_OF_VANILLA_MODULES=['Button','ComplicatedWires','Keypad','Maze','Memory','MorseCode','Password','SimonSays','WhosOnFirst','Wires','WireSequence']
+
+VANILLA_MODULES = {}
+MODDED_MODULES = {}
+
 def register_module(module):
     module_name=inspect.getmro(module)[0].__name__
     category = VANILLA_MODULES if module_name in LIST_OF_VANILLA_MODULES else MODDED_MODULES
     for identifier in module.identifiers:
         category[identifier] = module
 
-LIST_OF_VANILLA_MODULES=['Button','ComplicatedWires','Keypad','Maze','Memory','MorseCode','Password','SimonSays','WhosOnFirst','Wires','WireSequence']
-
-VANILLA_MODULES = {}
-MODDED_MODULES = {}
+from modules.base import Module, noparts, check_solve_cmd, gif_append, gif_output
```

Comment:
Please keep the comment that was here before.

Reply:
oh yes sorry